src/main/java/org/pac4j/j2e/filter/RequiresAuthenticationFilter.java

```diff
@@ -81,80 +72,71 @@ public void init(final FilterConfig filterConfig) throws ServletException {
             final Config config = ConfigBuilder.build(configFactoryParam);
             ConfigSingleton.setConfig(config);
         }
-
         this.clientName = getStringParam(filterConfig, Pac4jConstants.CLIENT_NAME, this.clientName);
-        CommonHelper.assertNotNull(Pac4jConstants.CLIENT_NAME, this.clientName);
-
-        this.requireAnyRole = getStringParam(filterConfig, Pac4jConstants.REQUIRE_ANY_ROLE, this.requireAnyRole);
-        this.requireAllRoles = getStringParam(filterConfig, Pac4jConstants.REQUIRE_ALL_ROLES, this.requireAllRoles);
         this.authorizerName = getStringParam(filterConfig, Pac4jConstants.AUTHORIZER_NAME, this.authorizerName);
 
-        this.useSessionForDirectClient = getBooleanParam(filterConfig, Pac4jConstants.USE_SESSION_FOR_DIRECT_CLIENT, this.useSessionForDirectClient);
-        this.allowDynamicClientSelection = getBooleanParam(filterConfig, Pac4jConstants.ALLOW_DYNAMIC_CLIENT_SELECTION, this.allowDynamicClientSelection);
+        // to help with backward compatibility
+        checkUselessParameter(filterConfig, "clientsFactory");
+        checkUselessParameter(filterConfig, Pac4jConstants.IS_AJAX);
+        checkUselessParameter(filterConfig, Pac4jConstants.STATELESS);
+        checkForbiddenParameter(filterConfig, Pac4jConstants.REQUIRE_ANY_ROLE);
```

Comment:
Block the use of these parameters as they would be ignored and no security would be applied.